velox/docs/configs.rst

```diff
@@ -389,6 +389,11 @@ Hive Connector
      - integer
      - 128MB
      - Maximum distance in bytes between chunks to be fetched that may be coalesced into a single request.
+   * - num_cached_file_handles
+     - integer
+     - 20000
+     - Maximum number of entries in the file handle cache. The value must be non-negative. Value zero
```

Comment:
> means no cache eviction

Not sure if readers will understand this. Perhaps, "Zero value indicates infinite cache capacity."

Reply:
Make sense, updated.